networkx/drawing/layout.py

```diff
@@ -983,6 +983,14 @@ def spiral_layout(G, scale=1, center=None, dim=2, resolution=0.35, equidistant=F
     --------
     >>> G = nx.path_graph(4)
     >>> pos = nx.spiral_layout(G)
+    >>> nx.draw(G, pos=pos)
+    
```

Comment:
@Lukong123 there is a <tab> here, it should be an empty line.

[Yes, I know we can't see it but this is one of the fun things about python and our linter `black` 😄 ]

Reply:
Wow, okay I'll take of note of that